wsdl/ver10/recording.wsdl

```diff
@@ -646,6 +644,212 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="MediaType" type="xs:string">
+						<xs:annotation>
+							<xs:documentation>As defined in RFC 6381 including section 3 Codecs.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="StoragePath" type="xs:string" >
+						<xs:annotation>
+							<xs:documentation>
+								The URI which combines the storage configuration StoragePath and the segment object path.
+							</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="Status" type="trc:UploadStatus">
+						<xs:annotation>
+							<xs:documentation>See trc:UploadStatus for allowed values.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:any namespace="##any" processContents="lax" minOccurs="0" maxOccurs="unbounded"/>	 <!-- first Vendor then ONVIF -->
+				</xs:sequence>
+				<xs:anyAttribute processContents="lax"/>
+			</xs:complexType>
+			<xs:element name="StopExportRecordedSegments">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="OperationToken" type="tt:ReferenceToken">
+							<xs:annotation>
+								<xs:documentation>Unique ExportRecordedSegments operation token</xs:documentation>
+							</xs:annotation>
+						</xs:element>
+					</xs:sequence>
+				</xs:complexType>
+			</xs:element>
+			<xs:element name="StopExportRecordedSegmentsResponse">
+				<xs:complexType>
+					<xs:sequence/>
+				</xs:complexType>
+			</xs:element>
+
+			<xs:complexType name="ExportRecordedSegmentStatus">
+				<xs:sequence>
+					<xs:element name="Segment" type="trc:UploadSegmentResult" minOccurs="0" maxOccurs="unbounded">
+						<xs:annotation>
+							<xs:documentation>Information about the segment(s) to be exported.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+					<xs:element name="HasMoreResults" type="xs:boolean" minOccurs="1">
+						<xs:annotation>
+							<xs:documentation>A value indicating that the export has reached the maximum count and a new request must be sent to query the export state of further segments.</xs:documentation>
+						</xs:annotation>
+					</xs:element>
+				</xs:sequence>
+			</xs:complexType>
+			<xs:simpleType name="UploadStatus">
+				<xs:restriction base="xs:string">
+					<xs:enumeration value="Pending"/>
+					<xs:enumeration value="Completed"/>
+					<xs:enumeration value="Failed"/>
+					<xs:enumeration value="Exporting"/>
+				</xs:restriction>
+			</xs:simpleType>
+
+			<xs:element name="GetExportRecordedSegmentState">
+				<xs:complexType>
+					<xs:sequence>
+						<xs:element name="OperationToken" type="tt:ReferenceToken">
+							<xs:annotation>
+								<xs:documentation>Unique ExportRecordedSegment operation token</xs:documentation>
+							</xs:annotation>
+						</xs:element>
+						<xs:element name="StartTime" type="xs:dateTime">
+							<xs:annotation>
```

Comment:
Shouldn't StartTime be optional? In the most simplistic case the client wants to get information about all segments.